### DISCUSSION TITLE
Velero backup creation stuck at Inprogress

### DISCUSSION THREAD

**Discussion by irparajababu** [vmware-tanzu/velero#6147] [Community support Q&A]
Hi Velero Team,

We are running the velero as kubernetes container sometimes it is creating the backups and sometimes it is stuck in Inprogress state but not moving completed or failed state. 

version details.
velero/velero: 1.6.3 

Please help us find out the root cause we tried to find out the solution but not giving any clue.

**Comment by qiuming-best** [comment]
you could try a higher version of velero, such as `v1.10.2`

**Reply by irparajababu** [reply]
Does upgrade resolve the above issue?

We will try to upgrade the version from 1.6.3 to 1.10.2

**Reply by medivyaa** [reply]
Hi ,

Do you find any solution for your issue
